nominal/core/run.py

```diff
@@ -210,6 +212,25 @@ def list_attachments(self) -> Sequence[Attachment]:
         """List a sequence of Attachments associated with this Run."""
         return list(self._iter_list_attachments())
 
+    def add_assets(self, assets: Iterable[Asset] | Iterable[str]) -> None:
```

Comment:
does it make sense to keep this a single asset for now and then change it when the backend properly supports multi-asset runs? Just so users don't trip up on this

Reply:
Yeah, can do. I missed the earlier message about multi-asset run support not being implemented imminently.